README.md

```diff
@@ -47,14 +47,24 @@ cd taskcluster-host-secrets
 npm install
 export TASKCLUSTER_SCOPE_BASE="assume:project:testing-host-secrets:host:"
 export TASKCLUSTER_CREDENTIALS_EXPIRE="1 day"
-export TASKCLUSTER_ALLOWED_IPS="\*"
+export TASKCLUSTER_ALLOWED_IPS="0.0.0.0/0"
 export TASKCLUSTER_CLIENT_ID=<snip>
 export TASKCLUSTER_ACCESS_TOKEN=<snip>
+export FORCE_SSL=true
+export TRUST_PROXY=false
 export PORT=8080
 export NODE_ENV=production
+export DEBUG="host-secrets:*"
 node lib/main server
 ```
 
+Unlike other taskcluster services which automatically publish api references
+and schemas, this service is deployed to potentially many hosts and thus needs
+a way to split deployment environments.  We're using `NODE_ENV` to control
+things around how the environment should work with tools like Express, and
+`DEPLOY_ENV` to pick which section of the `config.yml` or `user-config.yml`
+file to use
```

Comment:
It's so that we can make sure that all of our depended upon libraries (e.g. Express) are set up properly to be in Production mode (`NODE_ENV`) while allowing us to use a profile other than `production` in the `config.yml` file.  For deployment, we probably want to be able to store the non-secret parts of each deployment in the upstream config.yml file.

Maybe just `user-config.yml` and deleting the production profile from `config.yml` would be enough.

Publishing docs for this service is tough because unlike things like the queue which is a single cluster of exactly identical processes, we could end up having slightly different versions of this service running in different datacenters.  We'd need to upload docs per cluster.  Does taskcluster-lib-docs support that?  Maybe we could write up a static document for this service that explains how to interact with it?